hrs-portlets-webapp/src/main/webapp/WEB-INF/jsp/contactInfo.jsp

```diff
@@ -142,6 +142,12 @@
                      ${prefs['notice'][0]}
                   </p>
                 </c:if>
+                <p class='edit-notice'>
+                  Your preferred name in HRS will have to be updated separately
```

Comment:
Passive voice. :( A more user-centered version of this text might be "The Human Resources System (HRS) does not honor Madison preferred name, but you can [update your HRS-specific preferred name separately](https://my.wisc.edu/HRSPortlets/go?urlKey=Personal%20Information)".

Reply:
I think your point is very valid. The stakeholders were set on this specific language. For now, let's publish as is. We can come back around in the future on this page for more improvements if it fits in the priorities.